Solutions/Corelight/Analytic Rules/CorelightC2RepetitiveFailures.yaml

```diff
@@ -16,16 +17,15 @@ tactics:
   - CommandAndControl

```

Comment:
Hello @ekoyle relevant techniques is missing from this detection, Please add it below tactics: